src/gui/tray/activitylistmodel.cpp

```diff
@@ -694,8 +694,21 @@ void ActivityListModel::slotTriggerDefaultAction(const int activityIndex)
             ? InvalidFilenameDialog::InvalidMode::ServerInvalid
             : InvalidFilenameDialog::InvalidMode::SystemInvalid;
 
+#ifdef Q_OS_WIN
+        // Edge case time!
+        // QDir::filePath checks whether the passed `fileName` is absolute (essentialy by using `!QFileInfo::isRelative()`).
+        // On Windows, if `fileName` starts with a letter followed by a colon (e.g. "A:BCDEF"), it is considered to be an
+        // absolute path.
+        // Since the complete desired path is required by InvalidFilenameDialog, catch that special case here and prefix it ourselves...
+        const auto filePath = activity._file.length() >= 2 && activity._file[1] == ':'
+                              ? folder->path() + activity._file
```

Comment:
why not always using this for all platforms ?
would make the code simpler

Reply:
then maybe that should be made an utility method of `FolderMan::instance()->folder` such that we can ensure we have the same fix everywhere